docs/manual_full.tex

```diff
@@ -28,6 +31,11 @@
    margin=2.54cm
 }
 
+% pandoc packages
+\usepackage{lmodern} % Is it necessary? CM looks the same, doesn't it?
+% \usepackage{fixltx2e} % provides \textsubscript % Not necessary?
+% \usepackage[utf8]{inputenc} % Not necessary?
```

Comment:
Better to have -- if someone accidentally adds non-ascii characters this will make LaTeX understand them.